testcases/misc_tests/multi_instance.c

```diff
@@ -183,8 +183,8 @@ int main(int argc, char **argv)
     // generate a key pair for slot 1
     testcase_new_assertion();
     rv = do_GenerateTokenRSAKeyPair(slot_id1, session1,
-                                    (CK_BYTE *)"RSA-1024-SLOT1",
-                                    1024, &hPubKey1, &hPrivKey1);
+                                    (CK_BYTE *)"RSA-2048-SLOT1",
+                                    2048, &hPubKey1, &hPrivKey1);
```

Comment:
The same should be done for the fork test, I guess.

Reply:
Yep, right ... will do.

Reply:
BTW: in both cases we can leave the pub_exp as it is (0x10001). Smaller exponents would not be allowed in fips mode.